core/src/main/scala/akka/persistence/cassandra/journal/CassandraTagRecovery.scala

```diff
@@ -58,13 +59,10 @@ trait CassandraTagRecovery {
     rawEvent
   }
 
-  private[akka] def sendTagProgress(tp: Map[Tag, TagProgress], ref: ActorRef): Future[Done] = {
+  private[akka] def sendTagProgress(pid: String, tp: Map[Tag, TagProgress], ref: ActorRef): Future[Done] = {
     implicit val timeout = Timeout(10.second)
```

Comment:
perhaps make it a constant instead of creating new object each time (not implicit constant, though)